### PR TITLE
add variable to replace hardcode to deploy across multiple projects

### PR DESCRIPTION
**Pull request by lyleaf**

### CODE REVIEW

File: functions/index.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ const admin = require('firebase-admin');
 const path = require('path');
 
 admin.initializeApp();
-
-const BUCKET_NAME = 'barnard-project-audio'
+const projectId = admin.instanceId().app.options.projectId
```
Comment (parikhshiv) [inline functions/index.js]: Looks good, but the project id is still hardcoded on the frontend, in ApiUtils.js.

Comment (parikhshiv) [reply functions/index.js]: also, please add semicolons to your added lines in this file!

File: functions/index.js [hunk 2]
```diff
@@ -31,6 +31,9 @@ exports.saveAudioSuggestions = functions.https.onRequest(async (req, res) => {
       const metadata = await file.getMetadata();
       const mediaLink = metadata[0].mediaLink;
       console.log(`Audio available publicly at ${mediaLink}.`);
+      res.set('Access-Control-Allow-Origin', "*");
```
Comment (borismus) [inline functions/index.js]: return cors(...) should mitigate the need for this explicit `Access-Control-Allow-Origin` header.

File: functions/index.js (hunk 1 above)
Comment (lyleaf) [reply functions/index.js]: Done.

File: functions/index.js (hunk 2 above)
Comment (lyleaf) [reply functions/index.js]: Done.

File: functions/index.js [hunk 3]
```diff
@@ -40,34 +41,31 @@ exports.saveAudioSuggestions = functions.https.onRequest(async (req, res) => {
...
-  res.status(200).send("Translation suggestions saved.");
 });
 
 exports.addTranslations = functions.https.onRequest(async (req, res) => {
   return cors(req, res, async () => {
-    console.log('Add a translation');
-
-    //req content type should be application/json
-    snapshot = await admin.firestore().collection('translations').doc(req.body.english_word).set({
+    var snapshot = await admin.firestore().collection('translations').doc(req.body.english_word).set({
```
Comment (borismus) [inline functions/index.js]: nit: prefer let or const

File: functions/trix2firestore.py [hunk 4]
```diff
@@ -49,27 +33,24 @@ def main():
...
                 x = {
                       "english_word": english_word,
                       "translation": translation,
+                      "frequency": frequency,
                       "transliteration": transliteration,
                       "sound_link": sound_link
                     }
                 y = json.dumps(x)
-                the_file.write('\n')
                 headers = {'content-type': "application/json"}
```
Comment (borismus) [inline functions/trix2firestore.py]: As discussed yesterday, consider moving to the firestore python API.

**Pull request merged**